package.json

```diff
@@ -17,7 +18,8 @@
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "nodemon": "^3.0.1"
```

Comment:
Why you add nodemon ? this have to go in another PR

Reply:
It's ok but it should be in dev dependencies

Reply:
Yes...Sorry i didn't notice that